proposals/lip-0058.md

```diff
@@ -763,8 +763,6 @@ In the before application processing stage of a genesis block `b`, the BFT store
   * `bftVotes.maxHeightCertified = b.header.height`,
   * `bftVotes.blockBFTInfos` is an empty array,
   * `bftVotes.activeValidatorsVoteInfo` is an empty array.
-* The variable `genesisHeight` is initialized as follows:
```

Comment:
So, to make sure I completely follow the reasoning behind this change: According to old specs, this variable was initialized in the genesis block "before application processing" stage and should have been available in the context. 

However (correct me if I am wrong) this was not done in the implementation. So the reason of removing this is that is was not implemented, but somehow having this variable is not strictly necessary, since we can recover it from the BFT Votes substore after the genesis block?  

Removing this would imply that `genesisHeight` would not be available in the future (after `blockBFTInfos` does not contain the genesis block info anymore). But it seems that this is not necessary to have in any other part of the LIP.. 

Let me know if we are on the same page.

Reply:
Yep, the implementation did not use or initialize the genesis height, but instead was using `bftVotes.maxHeightPrevoted`. So the implementation is doing exactly what the same as the updated version of this LIP.